src/witan/validation.clj

```diff
@@ -15,12 +16,28 @@
 (defmulti category-valid?
   (fn [category lines] category))
 
+(defn all-years-starting-from-2011
+  [year-headers]
+  (let [years (sort (map #(java.lang.Integer/parseInt (re-find #"\d{4}" %)) year-headers))]
+    (=  (vec (range 2011 (inc (last years))))
+        years)))
+
 (defmethod category-valid? development-data-category
   [category lines]
-  (let [req-headers #{"GSS Code Borough" "Borough Name" "GSS Code Ward" "Ward Name"}
-        inc-headers (set (clojure.string/split (first lines) #","))]
-    (when-not (empty? (clojure.set/difference req-headers inc-headers))
-      (str "The header row of the file is missing one or more required rows. We expect " (clojure.string/join "," req-headers)))))
+  (let [req-headers #{"gss.code.borough" "gss.code.ward" "ward.name"}
+        inc-headers (str/split (first lines) #",")
+        formatted-headers (->> inc-headers
+                               (map str/lower-case)
+                               (map #(str/replace % #"\s" "."))
+                               (map #(str/replace % #"\"" ""))
+                               (set))
+        year-headers (filter #(re-find #"\d{4}-\d{4}" %) inc-headers)]
+    (cond
```

Comment:
it looks like this returns a truthy value if the headers are no valid which implies the naming of the fn is inverted?